docs/_data/atlantic-2023-schedule.yaml

```
@@ -0,0 +1,112 @@
+writing_day:
+  - time: '13:00'
+    title: 'Platform opens'
+    duration: '01:00'
+  - duration: '01:00'
+    title: Morning introduction - Introduction to Sprints & Project Introductions
+  - duration: '00:30'
+    title: Team up and start working
+  - duration: '01:30'
+    title: Welcome Wagon conference introduction and platform tour
+  - duration: '01:30'
+    title: '🍲 Lunch Break - Determine length with table'
```

It probably won't be lunch for the Eastern Hemisphere participants.
```suggestion
    title: '🍲 Meal Break - Determine length with table'
```

---

It's possible we can skip long meal breaks entirely, and let folks graze in the 'coffee' breaks for a more compressed day?